app/views/users/confirmation.html.erb

```diff
@@ -1,20 +1,20 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts("Almost Done!") %></h2>
+<h2 class="heading"><%= t(".page_heading") %></h2>
 <!--/descriptions-->
 <!--main content-->
 <div class="userstuff">
   <p>
-    <%= ts("You should soon receive a confirmation email at the address you gave us. This will have the link you need to follow in order to activate your account and complete the account creation process. The confirmation email will come from <strong>%{return_address}</strong> -- you might want to add this to your address book to make sure you get the email.", return_address: ArchiveConfig.RETURN_ADDRESS).html_safe %>
+    <%= t(".receive_email_html", return_address: ArchiveConfig.RETURN_ADDRESS) %>
   </p>
   <p>
-    <%= ts("If you haven't received this email within 24 hours, and you don't find our email in your spam filter or Social folder, please <a href='%{support_path}'>contact Support</a> for help.", support_path: new_feedback_report_path).html_safe %>
+    <%= t(".no_email_html", contact_support_link: (link_to t(".contact_support"), new_feedback_report_path)) %>
   </p>
   <p>
     <% # We do days_to_purge_unactivated * 7 because it's actually weeks %>
-    <%= ts("<strong>Important!</strong> You must confirm your email address within %{days_before_purge} days, or your account will expire. If this happens, you can sign up again using the same invitation.", days_before_purge: AdminSetting.current.days_to_purge_unactivated? ? (AdminSetting.current.days_to_purge_unactivated * 7) : ArchiveConfig.DAYS_TO_PURGE_UNACTIVATED).html_safe %>
+    <%= t(".must_activate_html", days_before_purge: AdminSetting.current.days_to_purge_unactivated? ? (AdminSetting.current.days_to_purge_unactivated * 7) : ArchiveConfig.DAYS_TO_PURGE_UNACTIVATED) %>
   </p>
   <p>
-    <%= link_to ts('Return to Archive front page'), root_path %>
+    <%= link_to t('.go_back'), root_path %>
```

Comment:
Could you switch this to
```suggestion
    <%= link_to t(".go_back"), root_path %>
```
for consistency with the rest? In theory Hound should be annoying about views as well, but I'm not sure how well it works.

Reply:
Weird that Hound didn't complain, thank you for noticing!